fastapi_filter/base/filter.py

```diff
@@ -68,7 +67,7 @@ def ordering_values(self):
 
     @validator("*", pre=True)
     def split_str(cls, value, field):  # pragma: no cover
-        ...
+        return value
```

Comment:
The reason I used `...` vs. a no-op operation was to make `BaseFilterModel` a loose "interface". I'm curious to understand why you need to have to use a no-op in your custom filters. If you don't plan on using one of the methods, you can just **not** use it, no? Could you provide a use case to illustrate the need of defining a no-op instead?